const.go

```diff
@@ -42,3 +44,7 @@ const (
 	// EventsTransportSSE activates stream events transport
 	EventsTransportSSE
 )
+
+func init() {
+	log.SetFlags(log.Ltime | log.Ldate | log.Lshortfile)
+}
```

Comment:
Does it mean, that if a user will set different flags they will be overwritten? Or if another library does similar thing (setting flags in `init`) - the behaviour will be unexpected?
I think I would set in `init` only the values which belong to `go-marathon`.